eo.txt

```diff
@@ -1394,9 +1396,13 @@ server_packet(Avatar, Remove)
 "Player has appeared in nearby view"
 server_packet(Players, Agree)
 {
-	break
-	struct CharacterMapInfo character
-	char = 1
+	struct NearbyInfo nearby
+}
+
+"NPC has appeared in nearby view"
+server_packet(Appear, Reply)
```

Comment:
`Appear` has been renamed to `MapInfo` it's packet definition is
```
"MapInfo for characters and/or npcs"
server_packet(MapInfo, Reply)
{
	char num_entities
	break // omitted if not sending back characters
	struct CharacterMapInfo characters[num_entities] // omitted if not sending back characters
	struct NPCMapInfo npcs[] // reads to end
}
```

Reply:
Thanks, I'll move this change into `MapInfo, Reply` then.